### PR TITLE
[23.2] Initial release notes  for 23.2

### PR DESCRIPTION
**Pull request by dannon**
Ref #16742

To do:
- [x] insert proper links to videos in user release notes
- [x] add proper alt text to images, video

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

File: doc/source/releases/23.2.rst [hunk 1]
```diff
@@ -0,0 +1,950 @@
...
+
+Enhancements
+-------------------------------
+
+.. major_feature
+
+
+.. feature
+
+* Make form repeat blocks reordarable
```
Comment (mvdbeek) [inline doc/source/releases/23.2.rst]: Those are PRs that already went into 23.1 ...

Comment (dannon) [reply doc/source/releases/23.2.rst]: Which ones?  They're all milestoned 23.2, and checking at least the first few (with `git branch -a --contains <blah>`) I don't see any issues.

**Comment by jdavcs** [comment]
How do we get correct links to embedded videos? Should they be published to the galaxyproject youtube channel?

**Comment by jdavcs** [comment]
Thank you, @hexylena, for publishing the videos!

**Review by jdavcs — APPROVED** [review]

**Review by jdavcs — CHANGES_REQUESTED** [review]
video links need editing; maybe rearrange the figures for workflow comments for a better flow.

**Review by jdavcs — APPROVED** [review]

**Review by martenson — APPROVED** [review]

**Comment by martenson** [comment]
looks great, thanks @jdavcs!

**Pull request merged**